core/src/main/java/org/apache/calcite/rel/mutable/MutableScan.java

```diff
@@ -36,11 +36,12 @@ public static MutableScan of(TableScan scan) {
   @Override public boolean equals(Object obj) {
     return obj == this
         || obj instanceof MutableScan
-        && rel.equals(((MutableScan) obj).rel);
+        && rel.getTable().getQualifiedName().equals(((MutableScan) obj).rel
```

Comment:
Should give an indentation here ?

Reply:
It's optional. I think it's clear either way. My IDE would not give further indentation.